client/src/ui/track/Searchbox.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react";
+import Autocomplete from "@mui/material/Autocomplete";
+import { TextFieldProps, TextField, Typography } from "@mui/material";
+
+/**
+ * A custom version of the Autocomplete component with the label designed similarly to the InputField
+ * @param props TextFieldProps
+ */
+export function ComboBox(props: TextFieldProps) {
+  const { required, label, placeholder } = props; // Extracting placeholder
+  const { style, ...restProps } = props;
+
+  return (
+    <div style={style}>
+      <div
+        style={{
+          display: "flex",
+        }}
+      >
+        <Typography
+          sx={{
+            textAlign: "start",
+            fontWeight: "bold",
+            display: "flex",
+          }}
+        >
+          {label}
```

Comment:
Isn't this redundant with the InputField? Can we use the component here instead of redoing it here again remove redundant work

Reply:
I removed it, thank you, it looks much better now.